dvc/dependency/base.py

```diff
@@ -1,19 +1,23 @@
-from dvc.exceptions import DvcException
+from dvc.output.base import (
+    OutputDoesNotExistError,
+    OutputIsNotFileOrDirError,
+    OutputIsStageFileError,
+)
 
 
-class DependencyDoesNotExistError(DvcException):
+class DependencyDoesNotExistError(OutputDoesNotExistError):
```

Comment:
Need to do this to make `BaseDependency` work. Otherwise, it will complain because `BaseOutput` has different exception type whereas here, it's different.

Reply:
Though, need to figure out how to fix the messages, right now I have broken it.